loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -594,6 +596,70 @@ module BABYLON.GLTF2 {
             var material = getMaterial(runtime, primitive.material);
             multiMat.subMaterials.push(material);
 
+            // Morph Targets
+            if (primitive.targets !== undefined) {
+                morphTargetManager = new BABYLON.MorphTargetManager();
+
+                for (var index2 = 0; index2 < primitive.targets.length; index2++) {

```

Comment:
I don't see any other index2 variables in the file. Did you mean previously in history?